### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by Hitansh-Shah**
@allcaps I have added a readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running 
+`node -v`   
+- Install all the dependencies     
+`yarn`  
+- Compile the frontend by running       
+`yarn start`    
+For production build run    
+`yarn start`
```
Comment (allcaps) [inline README.md:28]: yarn build

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running 
+`node -v`   
+- Install all the dependencies     
+`yarn`  
+- Compile the frontend by running       
+`yarn start`    
+For production build run
```
Comment (allcaps) [inline README.md:27]: Maybe: "Or, create a production build"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### Frontend
+
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running 
+`node -v`   
+- Install all the dependencies     
+`yarn`  
+- Compile the frontend by running
```
Comment (allcaps) [inline README.md:25]: Maybe: "Develop the frontend, run a file watcher."

File: README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/).
+
+## Development
+
+### Frontend
+
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running
```
Comment (allcaps) [inline README.md:21]: shoule isn't a word ;). I also like instructions to be as compact as possible. People do not read. Maybe: "Check the Node version number. It should be `v16.*`"

File: README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Wagtail guide brings you:
+- Tutorials, 
+- How-to, 
+- Reference 
+- Background information
+
+## Google Summer of Code
+
+This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/).
```
Comment (allcaps) [inline README.md:13]: Maybe link GSoC here too? https://summerofcode.withgoogle.com/programs/2022/projects/7nMw2hTq

File: README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+`yarn start`    
+
+### Backend
+- Verify that your python version is `3.9.*` by running     
+`python -V` 
+- Setup virtual environment     
+    ``` bash
+    python -m venv env
+    source env/bin/activate
+    ```
```
Comment (allcaps) [inline README.md:37]: I think the indent is not needed, is it?

**Review by allcaps — CHANGES_REQUESTED** [review]
Seems a lot, but they are all nitpicks.

General remark, maybe add some white lines? It makes the readme legible in plain text.

This is why I also  like code blocks (the indented style).
 https://www.jetbrains.com/help/hub/Markdown-Syntax.html#indented-code-blocks 

This way a dev can copy paste from plain text without getting the (*&^%$# quotes.

The ` `` ` are more suitable for inline code.

File: README.md (hunk 6 above)
Comment (Hitansh-Shah) [reply README.md:37]: If you want to align the code block with the the other text, then yes it is needed. If I remove the indent, the code block will shift to left and will not align with bullets as they are indented.

**Review by allcaps — APPROVED** [review]

**Pull request merged**